src/lib.rs

```diff
@@ -89,6 +89,8 @@ pub struct Record {
     pub alarm: Option<Alarm>,
     /// The UAS location, which may be given in one of several declaration types.
     pub location: Location,
+    /// The UAS velocity given in ENU coordinates (relative to the UAS and given in meters per second).
```

Comment:
What does "relative to the UAS" mean in this context? It's a velocity so it can't be relative to itself.

Reply:
Relative to the UAS means that the ENU coordinates are centered on the UASs position, if that makes sense. Let me know if this shouldn't be this way.

Reply:
That would make sense for a position, but I don't see how that would add any information to a velocity.